controllers/single_page/dashboard/system/environment/security_header_extended.php

```diff
@@ -14,6 +14,7 @@ public function view()
         $this->set('coop', $this->getPackageConfig()->get('security.cross_origin_opener_policy'));
         $this->set('coep', $this->getPackageConfig()->get('security.cross_origin_embedder_policy'));
         $this->set('accessControlAllowOrigin', $this->getPackageConfig()->get('security.access_control_allow_origin'));
+        $this->set('nosniff', $this->getPackageConfig()->get('security.x_content_type_options_nosniff'));
```

Comment:
@SashaMcr 

See other config keys. These do not include values.
It's `access_control_allow_origin` , not `access_control_allow_origin_none`
Please follow same naming rule